src/neuroconv/datainterfaces/text/excel/exceltimeintervalsinterface.py

```diff
@@ -7,5 +9,27 @@
 class ExcelTimeIntervalsInterface(TimeIntervalsInterface):
     """Interface for adding data from an Excel file to NWB as a TimeIntervals object"""
 
+    def __init__(
+        self,
+        file_path: FilePathType,
+        read_kwargs: Optional[dict] = None,
+        verbose: bool = True,
+    ):
+        """
+        Parameters
+        ----------
+        file_path : FilePath
+        read_kwargs : dict, optional
+            Passed to pandas.read_excel()
+        verbose : bool, default: True
+        """
+        read_kwargs = read_kwargs or dict()
+        super().__init__(file_path=file_path)
+        self.verbose = verbose
+
+        self._read_kwargs = read_kwargs
+        self.dataframe = self._read_file(file_path, **read_kwargs)
+        self.time_intervals = None
```

Comment:
This appears to be duplicated line for line from the parent - can you not simply call `super().__init__(file_path=file_path, read_kwargs=read_kwargs, verbose=verbose)`? If the only purpose is to give this a better non-inherited docstring

Reply:
good point